compiler/src/lnntoamm/Fn.ts

```diff
@@ -41,6 +42,37 @@ export default class Fn {
     this.body = body;
     this.metadata =
       metadata !== null ? metadata : new MetaData(scope, this.retTy);
+    while (
+      this.body.reduce(
+        (carry, stmt) => stmt.cleanup(this.scope) || carry,
```

Comment:
So we are now having the min Node version be 14, so we could do this instead:

```suggestion
        (carry, stmt) => stmt.cleanup(this.scope) ?? carry,
```

Reply:
the carry is a boolean though, so I think it's slightly more readable to keep it as `||`

Reply:
Ahh, I thought you were replacing a null-ish result from `cleanup` when I first wrote this. Keeping it as `||` is actually correct and `??` would break it.